test/integrationtest/projecttest/tests/graphql/testdata/cart_summary.graphql

```diff
@@ -0,0 +1,10 @@
+query {
+    Commerce_Cart {
+        cartSummary {
+            total: sumPaymentSelectionCartSplitValueAmountByMethods(methods: ["creditcard"]) {
```

Comment:
perhaps directly use the replacement pattern for the methods to make this reusable